mavros_extras/src/plugins/distance_sensor.cpp

```diff
@@ -242,7 +296,7 @@ DistanceSensorItem::Ptr DistanceSensorItem::create_item(DistanceSensorPlugin *ow
 		}
 
 		// optional
-		pnh.param("covariance", p->covariance, 0);
+		if (pnh.getParam("covariance", p->covariance) > 0) p->cov_is_def = true;
```

Comment:
But in calculations, covariance can get value of 0, so I think that is not the best approach.